addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -4,8 +4,11 @@
 
         <template id="report_picking">
             <t t-call="web.report_layout">
-                <div class="article o_report_layout_standard">
-                    <t t-foreach="docs" t-as="o">
+                <t t-foreach="docs" t-as="o">
+                    <div class="article o_report_layout_standard"
+                        t-att-data-oe-model="o and o._name"
+                        t-att-data-oe-id="o and o.id"
+                        t-att-data-oe-lang="o and o.env.context.get('lang')">
```

Comment:
Fix seems fine, but imitate what the previous `external_layout` template was doing before. I.e. the `div` should stay outside of the `<t t-foreach..>` because we still want to print multiple picking's picking operations within the same page. Try selecting multiple pickings in the list view and print it to see what I mean.